code/modules/modular_computers/file_system/programs/generic/file_browser.dm

```diff
@@ -83,7 +85,10 @@
 	switch(href_list["action"])
 		if("PRG_openfile")
 			. = 1
-			open_file = href_list["name"]
+			var/datum/computer_file/F = HDD.find_file_by_name(href_list["PRG_openfile"])
```

Comment:
Wrong href, you won't get the file ever and the if condition below will always return false

Reply:
Use "name" instead of "PRG_openfile"